test/functional/ui/wildmode_spec.lua

```diff
@@ -31,6 +31,21 @@ describe("'wildmenu'", function()
     ]])
   end)
 
+  it(':sign <tab> <space> hides wildmenu, selects design, when laststatus=2 #8453', function()
```

Comment:
what does "selects design" mean? Can just be:

    it(':sign <tab> <space> hides wildmenu #8453', function()

Reply:
Ok! I'll change it--it was originally there because pressing the `<space>` selects the first option from the wildmenu ('design').

I agree though--a bit long and confusing for a test name.